src/AuthAction.php

```diff
@@ -93,92 +86,89 @@ class AuthAction extends Action
      * }
      * ```
      *
-     * If this callback returns [[Response]] instance, it will be used as action response,
-     * otherwise redirection to [[cancelUrl]] will be performed.
+     * If this callback returns {@see ResponseInterface} instance, it will be used as action response,
+     * otherwise redirection to {@see cancelUrl} will be performed.
      */
-    public $cancelCallback;
+    private $cancelCallback;
     /**
      * @var string name or alias of the view file, which should be rendered in order to perform redirection.
      * If not set - default one will be used.
      */
-    public $redirectView;
+    private string $redirectView;
 
     /**
      * @var string the redirect url after successful authorization.
      */
-    private $_successUrl;
+    private string $successUrl;
     /**
      * @var string the redirect url after unsuccessful authorization (e.g. user canceled).
      */
-    private $_cancelUrl;
+    private string $cancelUrl;
 
 
     /**
      * @param string $url successful URL.
      */
-    public function setSuccessUrl($url)
+    public function setSuccessUrl($url): void
     {
-        $this->_successUrl = $url;
+        $this->successUrl = $url;
     }
 
     /**
      * @return string successful URL.
      */
-    public function getSuccessUrl()
+    public function getSuccessUrl(): string
```

Comment:
This and many other methods aren't used outside of the class so could be made private.